sqlmesh/cli/main.py

```diff
@@ -379,6 +379,14 @@ def rollback(obj: Context) -> None:
     obj.rollback()
 
 
+@cli.command("create_schema_file")
```

Comment:
Can we somehow tie the command name back to the concept of external models? Like it's not at all clear what's a schema file and why should I care. Can we call it something like add_external_models? Or register_external_models?